sdks/python/apache_beam/testing/data/vcf/valid-4.0.vcf

```diff
@@ -0,0 +1,23 @@
+##fileformat=VCFv4.0
```

Comment:
You need to update setup.py so that this new folder is also packaged in a distribution. (https://github.com/apache/beam/blob/92615ca4c4e09bc63c37a933102035040c23aac9/sdks/python/setup.py#L162) If you do not do this, unit tests ran from a distribution will fail. Or you can optionally skip tests when that happens.

Skipping some tests might be a good idea considering that these files are large relative to the < 1MB size of the current distribution.

Reply:
I think it's a good idea to not include these files in the distribution as you said. I changed the tests to skip if the directory of vcf test files is missing.